LondonBritishLibrary/orient/BLorient4611.xml

```diff
@@ -59,13 +58,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </msItem>
                                     <msItem xml:id="ms_i1.2.1.4">
                                         <locus from="89r" to="89v"/>
-                                        <title ref="LIT2257salota"/>
```

Comment:
also here, etc. put all titles back and reread the guidelines link i had provided, thank you